test/symlink.yaml

```diff
@@ -0,0 +1 @@
+test/test.yaml
```

Comment:
Can you add a test that checks that this symlink is followed if you `ko apply -f test/symlink.yaml` ?

Reply:
I tried something to test this but couldn't make it work but anyways I wanted to share the code with you to see what I missed or did wrong 🙋🏻‍♂️ maybe you can help me to fix this 🙏

Reply:
thanks a ton, @dentrax, he helped me to fix this. ❤️